llvm/test/CodeGen/PowerPC/scalar-rounding-ops.ll

```diff
@@ -214,6 +214,48 @@ entry:
 
 declare i64 @llvm.lround.i64.f64(double)
 
+define dso_local i32 @test_lround32(double %d) local_unnamed_addr {
```

Comment:
nit: We're passing a double into the intrinsic in this test. I think the name of this test might sound a bit too similar to the other one (`test_lroundf32`). Maybe we should update this to indicate something like f64 (but returning an i32)?

Anyway, this is just a picky nit of mine but if you don't feel it is necessary you can feel free to disregard.